phantomgamesTests/SearchGameViewModelTests.swift

```diff
@@ -28,6 +28,14 @@ final class SearchGameViewModelTests: XCTestCase {
         super.tearDown()
     }
     class MockSearchGameRepository: SearchGameRepositoryType {
+        func fetchAPIImage(completion: @escaping (phantomgames.SearchGameResult)) {
```

Comment:
```suggestion
        func fetchAPIImage(completion: @escaping SearchGameResult) {
```
Search how to import your project into your unit tests so that you can omit his